### PR TITLE
Bluetooth: controller: Adds RX framed flow to ISO AL

### PR DESCRIPTION
**Pull request by asmk-ot**
Adds reassembly of framed PDUs to ISO adaptation layer.
Reference times are also computed based on anchor points,
transmission latency components and a PDU specific time offset.

Signed-off-by: Asger Munk Nielsen asmk@oticon.com

### CODE REVIEW

**Review by cvinayak — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/controller/ll_sw/isoal.h
Comment (asmk-ot) [reply subsys/bluetooth/controller/ll_sw/isoal.h]: Change request implemented

File: subsys/bluetooth/controller/ll_sw/isoal.c
Comment (asmk-ot) [reply subsys/bluetooth/controller/ll_sw/isoal.c]: Change request implemented

File: subsys/bluetooth/controller/ll_sw/isoal.c
Comment (asmk-ot) [reply subsys/bluetooth/controller/ll_sw/isoal.c]: change implemented

File: subsys/bluetooth/controller/ll_sw/isoal.c
Comment (asmk-ot) [reply subsys/bluetooth/controller/ll_sw/isoal.c]: change implemented

File: subsys/bluetooth/controller/ll_sw/isoal.c
Comment (asmk-ot) [reply subsys/bluetooth/controller/ll_sw/isoal.c]: change implemented

File: subsys/bluetooth/controller/ll_sw/isoal.c
Comment (asmk-ot) [reply subsys/bluetooth/controller/ll_sw/isoal.c]: change implemented

File: subsys/bluetooth/controller/ll_sw/isoal.c
Comment (asmk-ot) [reply subsys/bluetooth/controller/ll_sw/isoal.c]: will consider change in subsequent PR

File: subsys/bluetooth/controller/ll_sw/isoal.c
Comment (asmk-ot) [reply subsys/bluetooth/controller/ll_sw/isoal.c]: Moved, will will consider namechange in subsequent PR

**Comment by asmk-ot** [comment]
@cvinayak thanks for the review, please revisit after I have addressed your comments

**Review by cvinayak — COMMENTED** [review]
Nearly there, few missed nitpicks still, otherwise LGTM.

**Comment by cvinayak** [comment]
@wopu-ot @mtpr-ot Could someone take over this PR.

**Comment by mtpr-ot** [comment]
> @wopu-ot @mtpr-ot Could someone take over this PR.

I'll do the last fixes today.

**Review by cvinayak — APPROVED** [review]

**Comment by cvinayak** [comment]
@mtpr-ot Need second review.

**Review by mtpr-ot — APPROVED** [review]
LGTM

**Comment by mtpr-ot** [comment]
Can we merge, or do we need more reviews?

**Review by thoh-ot — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/controller/ll_sw/isoal.c
Comment (thoh-ot) [inline subsys/bluetooth/controller/ll_sw/isoal.c:689]: This looks really weird, how is {} supposed to be used with case statements?

File: subsys/bluetooth/controller/ll_sw/isoal.c
Comment (thoh-ot) [inline subsys/bluetooth/controller/ll_sw/isoal.c:639]: It might be an unsupported case, but a malfunctioning peer could send this, and we should be able to survive.

File: subsys/bluetooth/controller/ll_sw/isoal.c
Comment (thoh-ot) [inline subsys/bluetooth/controller/ll_sw/isoal.c:658]: It might be an unsupported case, but a malfunctioning peer could send this, and we should be able to survive.

**Review by thoh-ot — CHANGES_REQUESTED** [review]

**Comment by mtpr-ot** [comment]
Decided to close the PR, as there are too many fixes already in the PR pipeline. which make this PR look incomplete.

**Pull request closed without merging**
